source/Ship.cpp

```diff
@@ -3533,37 +3533,65 @@ double Ship::TurnRate() const
 
 
 
-double Ship::Acceleration() const
+// Get maximum forward thrust. Use regular thruster and afterburner if allThrust is set.
+// Otherwise use regular thrusters if they are available and afterburner only if not.
+double Ship::ForwardThrust(bool allThrust) const
 {
 	double thrust = attributes.Get("thrust");
```

Comment:
```suggestion
	double thrust = allThrust ? attributes.Get("thrust") :  attributes.Get("thrust") + attributes.Get("afterburner thrust");
	return thrust ? thrust : attributes.Get("afterburner thrust");
```

I would prefer it like this

Reply:
I see where you want to go with that. I think it's a bit harder to grasp, as you now have 4 code paths (although one trivial one) instead of the needed 3 code paths. But I am fine with it if you prefer that.
But I guess the double space after the colon in your suggestion is not intended or do I miss something?

Reply:
lets wait on another reviewer with this one, I am not too sure now.